src/prototypes/email-signup/signup.stories.js

```diff
@@ -0,0 +1,15 @@
+import articleHeading from './example/example-a.twig';
+import footerForm from './example/example-b.twig';
+import './example/example.scss';
+
+export default {
+  title: 'Prototypes/Email Signup Form',
+  parameters: {
+    docs: { page: null },
+    paddings: { disabled: true },
```

Comment:
Ugh, it looks like they may have changed this attribute name at some point...

```suggestion
    paddings: { disable: true },
```